crates/aptos/src/node/mod.rs

```diff
@@ -486,3 +494,174 @@ impl CliCommand<serde_json::Value> for ShowValidatorSet {
+    /// To keep Testnet simple, we used one private key for aptos root account
+    /// To manually generate a keypair, use generate-key:
+    /// `cargo run -p generate-keypair -- -o <output_file_path>`
+    #[clap(long, default_value = "/opt/aptos/mint.key")]
+    pub mint_key_file_path: String,
+    /// Ed25519PrivateKey for minting coins
+    #[clap(long, parse(try_from_str = ConfigKey::from_encoded_string))]
+    pub mint_key: Option<ConfigKey<Ed25519PrivateKey>>,
+    /// Address of the account to send transactions from.
+    /// On Testnet, for example, this is a550c18.
+    /// If not present, the mint key's address is used
+    #[clap(long, parse(try_from_str = AccountAddress::from_hex_literal))]
+    pub mint_account_address: Option<AccountAddress>,
+    /// Chain ID of the network this client is connecting to.
+    /// For mainnet: "MAINNET" or 1, testnet: "TESTNET" or 2, devnet: "DEVNET" or 3,
+    /// local swarm: "TESTING" or 4
+    /// Note: Chain ID of 0 is not allowed; Use number if chain id is not predefined.
+    #[clap(long, default_value = "TESTING")]
+    pub chain_id: ChainId,
+    /// Amount of coins to mint
+    #[clap(long)]
+    pub amount: u64,
+    /// Addresses of accounts to mint coins to, split by commas e.g. 0x1337,0x2,3
+    #[clap(long, group = "account-group")]
+    pub accounts: Option<String>,
+    /// File of addresses of account to mint coins to.  Formatted in YAML
+    #[clap(long, group = "account-group", parse(from_os_str))]
+    pub account_file: Option<PathBuf>,
+}
+
+#[async_trait]
+impl CliCommand<String> for MintCoins {
+    fn command_name(&self) -> &'static str {
+        "MintCoins"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<String> {
+        let mint_account_address = self.mint_account_address.unwrap_or_else(aptos_root_address);
+        let mint_key = if let Some(ref key) = self.mint_key {
+            key.private_key()
+        } else {
+            EncodingType::BCS
+                .load_key::<Ed25519PrivateKey>("mint key", Path::new(&self.mint_key_file_path))
+                .unwrap()
+        };
+        let faucet_account = LocalAccount::new(mint_account_address, mint_key, 0);
+        let service = Service::new(self.server_url, self.chain_id, faucet_account, None);
+
+        let accounts: HashSet<AccountAddress> = if let Some(accounts) = self.accounts {
+            accounts
+                .trim()
+                .split(',')
+                .map(|str| load_account_arg(str).unwrap())
+                .collect()
+        } else if let Some(path) = self.account_file {
+            let strings: Vec<String> =
+                serde_yaml::from_str(&std::fs::read_to_string(path.as_path()).unwrap()).unwrap();
+            strings
+                .into_iter()
+                .map(|str| load_account_arg(&str).unwrap())
+                .collect()
+        } else {
+            panic!("Either --accounts or --account-file must be specified");
```

Comment:
This function returns a result, can we do that instead of panic? Where possible we should avoid panicking, unwrapping, etc, it's not a good user experience.